### DISCUSSION TITLE
What the hell???

### DISCUSSION THREAD

**Discussion by www-MKRD-info** [mifi/lossless-cut#1132] [General]
What the hell???
EVERY SIX MONTHS for the last few years I needed such a lossless cutter / trimmer, and I have tried to use Gurgle search, to see if anything like this exists.
Every moron out there tried to lead me to the stupid workflow of installing OpenShot et al, importing, saving a "project" file, cutting, exporting, etc. And export settings NEVER match actual file settings of course!!! Heck, the new file is WORSE in quality but LARGER in size 10,000% of the time!!!

I just found this awesome project today... No explanation why Gurgle decided to finally let me know this exists, even though I use just about the same search terminology every six months.
Don't believe me??? I was holding out for a "VLC simple video editor" for years now, thinking that is the only team who would make a "project-less / dead-simple" video "editor". They promised me bullcrap vaporware, and nobody told me this software existed all this time??????????????????????????????

To be entirely honest, at first I even thought this is some crippleware. And I try it... And it actually works!!! It works intuitively!!! It does not even ask to be installed! It "exports" using autodetected settings from source file. What??? That is possible??????

I am going to donate to you. This blows all other crappy software like OpenShot (OpenShot for mainly being overkill for 95% of its users) out the water. 99.999999999999999999999999% of people need an "editor" such as this, just to trim beginning / end / chunks in middle, maybe splice in audio, mute or bleep a second or two, etc. What the hell is everyone leading everybody else into this moronic, idiotic workflow of using transcoding editors???

AWESOME!!! THANKS!!!

BUT I AM ALSO MAD THAT I HAVE NOT FOUND THIS YEARS EARLIER!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! WHAT THE FRACK GURGLE, HOW MANY TIMES HAVE I TRIED TO FIND EXACTLY THIS SOFTWARE???????????????????????????????????????????????????????????????????

**Comment by mifi** [comment]
thanks for your eccentric feedback! I guess the takeaway is that I should improve the SEO of losslesscut to make it easier for people to find...

**Reply by www-MKRD-info** [reply]
> thanks for your eccentric feedback! I guess the takeaway is that I should improve the SEO of losslesscut to make it easier for people to find...

Yes!!!
Except all of the sudden, you are high enough in Gurgle ratings with me trying to remember what search phrases I used to NOT find you before... Maybe it's my choice of search phrase today... As with modern Gurgle searches, one character off, and you get completely useless search results. We are back in 2000-era internet search results!!! Internet search indexers were meant for "techie people" "who know what they are doing" from 2000 to about 2015, then drastically every website went into "we gotta help the morons out there" mode, where they try to second-guess the meaning of my search query, while doing their best to redirect me towards purchase of sponsored Chinese junk... Thus, it no longer matters what you type into a search engine, and indeed it does not even matter what search engine or what website is used, they are ALL equally disgustingly bad after 2015 or so...

You are doing good, keep doing what you are doing!!! When my biweekly paycheck arrives, I will donate to your project! You remind me of FreeFileSync in open-source (FOSS) software category, software that is so awesome I am having a hard time believing it is made available to us, and how well the single individual maintains and improves the product...

**Comment by mattack1** [comment]
I know I've commented elsewhere, but my main "issue" is that it doesn't do MPEG2 files natively, so it's much more of a slog jump a while, wait for a preview frame, etc..  Basically everything I edit is MPEG2 files I download from my Tivo.. to keep the music segments from talk shows/SNL/etc.. 

but yeah, this reaction was kind of funny.
